mmdet/apis/train.py

```diff
@@ -77,19 +78,24 @@ def train_detector(model,
             seed=cfg.seed) for ds in dataset
     ]
 
-    # put model on gpus
-    if distributed:
-        find_unused_parameters = cfg.get('find_unused_parameters', False)
-        # Sets the `find_unused_parameters` parameter in
-        # torch.nn.parallel.DistributedDataParallel
-        model = MMDistributedDataParallel(
-            model.cuda(),
-            device_ids=[torch.cuda.current_device()],
-            broadcast_buffers=False,
-            find_unused_parameters=find_unused_parameters)
+    map_location = 'default'
```

Comment:
What does "default" mean? Are all cases covered?

- CPU -> CPU
- CPU -> GPU
- GPU -> CPU
- GPU -> GPU

Reply:
By default use cuda device https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/base_runner.py#L293.
I did not test 'cpu - gpu', but other works correctly.